OpenRVS/classes/OpenMultiPlayerWidget.uc

```diff
@@ -308,22 +329,219 @@ function Refresh(bool bActivatedByUser)
 	if ( ( m_LanServers.m_ClientBeacon == none ) || ( !m_LanServers.m_ClientBeacon.IsA('OpenClientBeaconReceiver') ) )
 		return;
 
+	//1.5 - if initiated by user, clear the list of current queries and start fresh
+	if ( bActivatedByUser )
+		OpenQueries.remove(0,OpenQueries.length);
 	//0.9: get each server in the list, then query for more info
+	//1.5 - query ServerList instead
+	j = 0;
+	while ( j < ServerList.length )

```

Comment:
Instead of this:
```
local int j;
j = 0;
while (j < ServerList.Length)
    j++;
```
What about this?
```
local int j;
for (j=0; j<ServerList.Length; j++)
```
Slightly more concise, and `for` loops with explicitly-defined bounds feel a bit safer than `while` loops.

Could just be the result of me primarily working in a language that only has `for` loops, haha.